packages/postcss-normalize-whitespace/src/index.js

```diff
@@ -19,8 +19,9 @@ function reduceWhitespaces(node) {
   } else if (node.type === 'div') {
     node.before = node.after = '';
   } else if (node.type === 'function') {
-    node.before = node.after = '';
-
+    if (node.value.toLowerCase() !== 'var') {
```

Comment:
We should keep spaces in `var`and `env` functions

Reply:
should I add this for `env` also then ?

```js
&& node.value.toLowerCase() !== 'env'

```

Reply:
Yes, better use `['var', 'env'].includes(node.value.toLowerCase())`

Reply:
yeah ok !